### PR TITLE
Add header, nav-bar, and style.

### PR DESCRIPTION
**Pull request by Richie9000**
Added the style in css to the navbar and the content of the page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,5 +7,40 @@
...
   </head>
-  <body></body>
+  <body>
+    <div class="wrapper">
+      <header class="top">
+        <a class="active" href="#">Logo</a>
+        <nav class="flex-nav">
+          <a href="#" class="toggleNav">☰</a>
+        </nav>
+      </header>
```
Comment (descholar-ceo) [inline index.html]: - [ ] This part should use the font and colors (background and font) defined for the template

File: index.html [hunk 2]
```diff
@@ -7,5 +7,40 @@
...
+              I’m a software developer! I can help you build a product, feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+          </div>
+          <div class="title2">
+            <p>LET'S CONNECT</p>
+          </div>
+        </section>
```
Comment (descholar-ceo) [inline index.html]: - [ ] The section has the proper background image (decorations are always background images, not img tags)

File: index.html [hunk 3]
```diff
@@ -7,5 +7,40 @@
...
+
+        <div class="icons">
+          <img src="/img/twitter.svg" alt="Twitter logo" />
+          <img src="/img/github.svg" alt="github logo" />
+          <img src="/img/Linkedin.svg" alt="Linkedin logo" />
+          <img src="/img/look.svg" alt=" logo" />
+          <img src="/img/medium.svg" alt="medium logo" />
+        </div>
```
Comment (descholar-ceo) [inline index.html]: - [ ] This section tag should cover all of these icons as well

Comment (descholar-ceo) [inline index.html]: - [ ] The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag)

File: index.html [hunk 4]
```diff
@@ -7,5 +7,40 @@
...
+          <div class="para">
+            <p>
+              I’m a software developer! I can help you build a product, feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+          </div>
+          <div class="title2">
+            <p>LET'S CONNECT</p>
```
Comment (descholar-ceo) [inline index.html]: - [ ] This `LET'S CONNECT` should be either a button or a link which comply with the following requirement: `The button (if the template has one) is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design)`

File: index.html [hunk 5]
```diff
@@ -7,5 +7,40 @@
...
+          </div>
+        </section>
+
+        <div class="icons">
+          <img src="/img/twitter.svg" alt="Twitter logo" />
+          <img src="/img/github.svg" alt="github logo" />
+          <img src="/img/Linkedin.svg" alt="Linkedin logo" />
+          <img src="/img/look.svg" alt=" logo" />
+          <img src="/img/medium.svg" alt="medium logo" />
```
Comment (descholar-ceo) [inline index.html]: - [ ] These icons images are broken. To fix  that, I would recommend you to write correct location path, make sure that all of the links start by a dot, eg: bad: ~src="/img/twitter.svg" alt="Twitter logo"~, good: `src="./img/twitter.svg" alt="Twitter logo"`

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @Richie9000 :handshake: :handshake: :handshake:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._
- [ ] The overall view of your portfolio is not looking like the given template:
screencapture-rawcdn-githack-Richie9000-Portfolio-setup-5437335600b5c717fec8550e62787a49e896245c-index-html-2021-06-03-19_42_18

=> Kindly re-visit [this template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) and adjust your portfolio accordingly

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Richie9000) [reply index.html]: I don't understand, what should I do in this section?

Comment (descholar-ceo) [reply index.html]: Hello @Richie9000 , your portfolio was actually supposed to have a background image, which makes it way similar to the given template found [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122), currently your project seems to not have any background image at all, 
screencapture-rawcdn-githack-Richie9000-Portfolio-setup-5437335600b5c717fec8550e62787a49e896245c-index-html-2021-06-03-19_42_18
That is how It looks like, without any appropriate background image, your project's live version is [here](https://rawcdn.githack.com/Richie9000/Portfolio-setup/5437335600b5c717fec8550e62787a49e896245c/index.html) kindly take a look at how it looks now and adjust it accordingly to what the template looks like

File: index.html (hunk 1 above)
Comment (DeliceLydia) [reply index.html]: This was resolved very well!

File: index.html (hunk 2 above)
Comment (DeliceLydia) [reply index.html]: It is good to see that you added a background image

File: index.html (hunk 3 above)
Comment (DeliceLydia) [reply index.html]: Now the section tag cover icons as well. well done!

Comment (DeliceLydia) [reply index.html]: This was addressed well.

File: index.html (hunk 4 above)
Comment (DeliceLydia) [reply index.html]: - [ ] please follow the design while working on this button refer to the previous reviews to make it identical to the design.

File: index.html (hunk 5 above)
Comment (DeliceLydia) [reply index.html]: Icon images are showing well now

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hello @Richie9000 ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments i added on the previous review to make changes needed._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Consider:

- To follow the given design by removing text-decoration on the logo and hamburger button
-
des
try to follow the design your name and "glad to see you are not on the same line"

### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (Richie9000) [reply index.html]: Hi Delice, in my template there's no button, is <a> link, what else should I do?

Comment (descholar-ceo) [reply index.html]: - [ ] Hello @Richie9000 , what you you need to do here is to style that link and make it look like this: Screenshot from 2021-06-04 06-03-57 from the template

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @Richie9000 :handshake: :handshake: , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._
- [ ] We still need you to style your portfolio and make sure that it looks like the given template, this is a hard requirement in order to consider this project as complete, the following is a table which compares your portfolio version and the given template,

|Your version|The template|
|---|---|
|Screenshot from 2021-06-04 06-06-04|Screenshot from 2021-06-04 06-07-00|
- [ ] Kindly make sure that you use the font and colors (background and font) defined for the template
- [ ] All of the images renders well, like [this image](https://github.com/Richie9000/Portfolio-setup/blob/ebcb9d0ca96214eec284e0d79bb1c71081afe9a5/index.html#L61) is broken, I think you should fix it, 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shaher-11 — APPROVED** [review]
Hello @Richie9000,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle: 

## Optional Suggestions 💡
_The following points are optional to implement however they're highly recommended!_:blush:

- Kindly increase the size of the burger menu icon
- Add a screenshot of your project to the readme file.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**